lighthouse-core/config/default.js

```diff
@@ -240,7 +241,7 @@ module.exports = {
         {id: 'dom-size', weight: 0, group: 'perf-info'},
         {id: 'critical-request-chains', weight: 0, group: 'perf-info'},
         {id: 'user-timings', weight: 0, group: 'perf-info'},
-
+        {id: 'error-logs', weight: 0, group: 'perf-info'},
```

Comment:
let's put this in Best Practices instead. We have no idea if its a perf concern, but a console clean-bill-of-health seems like a general best practice more any anything specific to perf.